habitat-lab/habitat/articulated_agents/mobile_manipulator.py

```diff
@@ -41,32 +38,21 @@ class MobileManipulatorParams:
     :property arm_joints: The joint ids of the arm joints.
     :property gripper_joints: The habitat sim joint ids of any grippers.
     :property wheel_joints: The joint ids of the wheels. If the wheels are not controlled, then this should be None
-    :property arm_init_params: The starting joint angles of the arm. If None,
-        resets to 0.
-    :property gripper_init_params: The starting joint positions of the gripper. If None,
-        resets to 0.
-    :property ee_offset: The 3D offset from the end-effector link to the true
-        end-effector position.
+    :property arm_init_params: The starting joint angles of the arm. If None, resets to 0.
```

Comment:
Wrapping the docstring like it was before shouldn't cause any problems for the doc generator. The successive lines just need to be indented respective to the first `:property :` line.